src/support/README.md

```diff
@@ -30,9 +30,9 @@ users can be found.
 
 There are also several CheriBSD email lists available:
 
-- CheriBSD announcement mailing list (low volume) **XXX: To update**
-- CheriBSD general discussion **XXX: To update**
-- CheriBSD security vulnerabilities contact address **XXX: To update**
+- [CheriBSD announcement mailing list (low volume)](https://lists.cam.ac.uk/sympa/info/cl-cheribsd-announce)
+- [CheriBSD general discussion](https://lists.cam.ac.uk/sympa/info/cl-cheribsd-discuss)
+- CheriBSD security vulnerabilities contact address [cl-cheribsd-security@lists.cam.ac.uk](mailto:mailto:cl-cheribsd-security@lists.cam.ac.uk) 
```

Comment:
```suggestion
- CheriBSD security vulnerabilities contact address [cl-cheribsd-security@lists.cam.ac.uk](mailto:cl-cheribsd-security@lists.cam.ac.uk) 
```

Reply:
Also feels like there should be a : or - or similar, looks weird at the moment. That or hyperlink the text like we do for the mailing lists, doesn't match at the moment.

Reply:
This second point was never resolved